README.md

```diff
@@ -4,6 +4,10 @@
 - waffle util tools
 - [Waffle Data Convention](https://snuailab.notion.site/Waffle-Data-Convention-7547fda8c1ca48798d00bd4658ea96bf)
 
+# Install
+- python >= 3.9
```

Comment:
Did you check the compatibility for all the versions(3.9, 3.10, 3.11)? Or, are you sure that all the features we are going to  implement from now will have no conflict with all the versions as well? I'm doubtful.

Reply:
It will be checked when ci tests run.
It is compatible with 3.9, 3.10 for now (maybe 3.11 not sure, i'll add 3.11 to ci test.)

And about your concern, we should implement new features compatible with that versions.
So, it doesn't matter.

Reply:
+ 3.11 is also compatible
commit c7be140